server/src/server.ts

```diff
@@ -268,7 +268,7 @@ let openedFile = (fileUri: string, fileContent: string) => {
         filesDiagnostics: {},
         namespaceName:
           namespaceName.kind === "success" ? namespaceName.result : null,
-        rescriptVersion: utils.findReScriptVersion(projectRootPath),
+        rescriptVersion: utils.findReScriptVersionForProjectRoot(projectRootPath),
```

Comment:
This is the root of the problem really.
Here my project is called `ronnies.be`, which is mistaken for a file.
At this point, we know it is a folder, so we can shortcut things I think.